db_master_deploy/deploy.sh

```diff
@@ -361,8 +392,6 @@ for source_object in "${array_source[@]}"; do
     array=()
 done
 
-# check for lockfile, if there is one exit script, lock file is created by import_databases.sh
-(ls ${lockfile} &> /dev/null) && { echo "lockfile found: ${lockfile} '$(cat ${lockfile})'" >&2; exit 1; }
```

Comment:
that's possible for a long time already. parallel deploys have been introduced with https://github.com/geoadmin/deploy/commit/0796b4516f41a64266feb214cda105422070cc97